tests/classify_test.py

```diff
@@ -1,6 +1,7 @@
 import contextlib
 import os.path
 import sys
+from zipfile import ZipFile
```

Comment:
(tiny) personally I'd `import zipfile` and use `zipfile.ZipFile` as the other modules are in this

Reply:
Switched to `import zipfile`.